exporters/newrelic/src/main/java/io/opentelemetry/exporters/newrelic/SpanBatchAdapter.java

```diff
@@ -103,9 +104,20 @@ private static Attributes generateSpanAttributes(Span span) {
           break;
       }
     }
+    addResourceAttributes(span, attributes);
     return attributes;
   }
 
+  private static void addResourceAttributes(Span span, Attributes attributes) {
```

Comment:
If you have this method return `Attributes` then you can save a line in the caller (and it might read a slight bit nicer as well).

Reply:
I refactored this; give it a look